Zeroconf/ZeroconfResolver.cs

```diff
@@ -37,13 +37,15 @@ static IEnumerable<string> BrowseResponseParser(Response response)
                     (address, buffer) =>
                     {
                         var resp = new Response(buffer);
-                        Debug.WriteLine($"IP: {address}, Bytes: {buffer.Length}, IsResponse: {resp.header.QR}");
+                        var firstPtr = resp.RecordsPTR.FirstOrDefault();
+                        var dn = firstPtr?.PTRDNAME.Split('.')[0] ?? "N/A";
```

Comment:
Instead of N/A, would it make more sense to use an empty string? Then below, as the dict key, only add the :dn if it's not empty?

Reply:
I had made the assumption that the DisplayName would never be empty if QR was true, is that not the case? It looked a little more confusing doing the check to omit the name if it was blank, but take a look and see if you like that better. Thanks for reviewing so quick!